### PR TITLE
move from pages/integrations to models collection

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Notes for Reviewers**

This PR 
- removes all the pages/integrations markdown files
- initializes `models` collection, which has all the models' metadata in frontmatter variables.
> models collection can be accessed through `site.models` variable and filepath is : `docs/_models`
- updates site.pages with site.models for counting total number of integrations.
- updates integration layout and styles to show components, if available

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 3750b64ccdd64d14265353a0e9cf7a6df26878a6 at: https://66177964b039b441ba09e9d7--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 7cad010a631c24c9b0a9457645303e6aa23e5845 at: https://661788f4e250554ec8f25f7d--meshery-docs-preview.netlify.app

File: docs/_layouts/integration.html [hunk 1]
```diff
@@ -1,8 +1,37 @@
...
+        </h3>
+        {% endif %}
+    </section>
+
+    <section class="componentsSection">
+        {% for component in page.components %}
+        {% if component.name %}
+        <div class="maincontainer">
+            <div class="componentimg">
+                <img id="logo-dark-light" src="/{{ component.colorIcon }}" alt="{{ component.name }}"  data-logo-for-dark="/{{ component.whiteIcon }}"
```
Comment (vishalvivekm) [inline docs/_layouts/integration.html:23]:
```suggestion
                <img id="logo-dark-light" src="/{{ component.colorIcon }}" alt="{{ component.name }}"  data-logo-for-dark="/{{ component.whiteIcon }}"
```
```suggestion
                <img id="logo-dark-light" src="/{{ component.whiteIcon }}" alt="{{ component.name }}"  data-logo-for-dark="/{{ component.whiteIcon }}"
```

**Comment by l5io** [comment]
🚀 Preview for commit 633a34d5dde957e01040774835daa0a13a83c989 at: https://66178aa6e4aa374f730ae606--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 7cad010a631c24c9b0a9457645303e6aa23e5845 at: https://66178d5fb039b451c309e7ea--meshery-docs-preview.netlify.app

**Pull request merged**